front/components/agent_builder/AgentBuilderPreview.tsx

```diff
@@ -130,11 +125,7 @@ export function AgentBuilderPreview() {
   };
 
   return (
-    <div
-      className="flex h-full w-full flex-col"
-      role="main"
-      aria-label="Agent preview"
-    >
+    <div className="flex h-full w-full flex-col" aria-label="Agent preview">
```

Comment:
There is a content flickering right now, maybe we should show a spinner instead? 

https://github.com/user-attachments/assets/603c49b4-32bb-494a-a43a-b93e307f63f7

Reply:
I don't seem to have flickering on my end

Reply:
You do have the flickering right? Are all talking about the flash message "unable..."? I have it too.